### PR TITLE
fix(cli): Apply non destructive changes

### PR DESCRIPTION
**Pull request by byawitz**
## What does this PR do?
Apply non-destructive changes

image

### CODE REVIEW

File: templates/cli/lib/commands/push.js.twig [hunk 1]
```diff
@@ -1063,7 +1212,7 @@ const pushCollection = async ({ returnOnZero } = { returnOnZero: false }) => {
             attributes = await attributesToCreate(collection.remoteVersion.attributes, collection.attributes, collection);
 
             if (Array.isArray(attributes) && attributes.length <= 0) {
-                log(`No changes has been detected. Skipping ${collection.name} ( ${collection['$id']} )`);
+                log(`No destructive changes has been detected. Skipping ${collection.name} ( ${collection['$id']} )`);
```
Comment (Meldiron) [inline templates/cli/lib/commands/push.js.twig:1215]: Do we have some non-destructive ones?

Comment (Meldiron) [reply templates/cli/lib/commands/push.js.twig:1215]: If so, we shouldnt skip them. we should do them.
Also, in databases specifically, I would ALWAYS ask for confirmation, so they always see the table and can decide if all those changes should be applied - even if non-destructire. It's like having dry-run.

Comment (byawitz) [reply templates/cli/lib/commands/push.js.twig:1215]: The skipping is on the destructive changes, the non-destructive changes are being applied (as max)

Comment (byawitz) [reply templates/cli/lib/commands/push.js.twig]: image

**Pull request merged**